wpeverest/user-registration
# Feature - Success message display position option

### All Submissions:






### Changes proposed in this Pull Request:
Previously by default the success message after successful registration was always displayed at the bottom of the form. This PR provides an option in individual form settings that provides the admin with the flexibility to choose a position i.e Top or Bottom of the form to display the success message.

### How to test the changes in this Pull Request:

1. Navigate to the form builder of any form and under the General Settings section of Form Settings tab, there is an option labelled `Success message position` as shown in the attached image.
2. Select the option i.e Top to Bottom as per your need.
3. In frontend verify if the messages are being displayed in the appropriate position.

![success-message](https://user-images.githubusercontent.com/23380477/101599959-c1d20e00-3a22-11eb-9548-26d9c6436d1f.PNG)

### Types of changes:




### Other information:




### Changelog entry

> Feature - Success message display position option
